enhancing/enhancer.spec.js

```diff
@@ -1,2 +1,77 @@
 const enhancer = require('./enhancer.js');
 // test away!
+
+it('item => new item with durability === 100', () => {
+  const received = {
+    name: 'Sword of Regret',
+    enhancement: 12,
+    durability: 28
+  };
+  const expected = {
+    name: 'Sword of Regret',
+    enhancement: 12,
+    durability: 100
+  };
+  expect(enhancer.repair(received)).toEqual(expected);
+});
+
+it('item => new item with enhancement + 1 (up to a max of 20)', () => {
```

Comment:
there's nothing wrong with this, but one of the nice things about the jest `it` pattern is that it lends itself to very English-readable test descriptions--it makes it slightly more readable if this were to say `it('increments the item's enhancement by 1 (max 20)` for example. another reason while it's a good idea to also wrap your functions' `it` statements in a `describe` wrapper is you can say exactly which function you're testing here, instead of having to read it in the actual test below, ex: `describe("succeed()", ()=>{
it("should do stuff", ()=> {
test
})
})`